server/util_test.go

```diff
@@ -0,0 +1,43 @@
+package server
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/wire"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+//func AppendJSON(cdc *wire.Codec, baseJSON []byte, key string, value json.RawMessage) (appended []byte, err error) {
```

Comment:
Comment here so we know the method signature or unintentional?

Reply:
no thanks, I'll delete